### PR TITLE
chore: tag 0.4.3

### PR DESCRIPTION
**Pull request by ncloudioj**
r? @jbuck

### CODE REVIEW

**Comment by jbuck** [comment]
looks like this is failing some tests, and I don't think I can actually review this patch since it'll require review from a code owner

**Comment by ncloudioj** [comment]
> I don't think I can actually review this patch since it'll require review from a code owner

Right, nvm. The dev build succeeded, I think we can use that for QA for now. 

> looks like this is failing some tests

Yeah, that was a known intermittent failure (#250) with the dependency in Kinto.

**Review by pjenvey — APPROVED** [review]

**Pull request merged**
